modules/android/examples/codesnippet_collection.kt

```diff
@@ -120,6 +120,22 @@ class BasicExamples(private val context: Context) {
         database.delete()
     }
 
+    // ### Database FullSync
+    fun DatabaseFullSyncExample() {
+        // tag::database-fullsync[]
+        val database = Database(
+            "my-db",
+            DatabaseConfigurationFactory.newConfig(
+                fullSync = true
+            )
+        ) 
+        // end::database-fullsync[]
+        
+        config.setFullSync()
+        
+    }
```

Comment:
I am unsure of how correct this is. I would wait on Blake to review Android/Kotlin

Reply:
> config.setFullSync()

This is probably wrong.

Reply:
This is correct.  It is also cute.  Can we please keep it?